src/PSAppDeployToolkit/Public/Show-ADTInstallationWelcome.ps1

```diff
@@ -103,6 +104,12 @@ function Show-ADTInstallationWelcome
     .PARAMETER RequiredDiskSpace
         Specify required disk space in MB, used in combination with CheckDiskSpace.
 
+    .PARAMETER Subtitle
+        Subtitle of the prompt. Overrides Subtitle as defined in Strings.psd1.
+
+    .PARAMETER Title
+        Title of the prompt. Overrides $adtSession.InstallTitle
```

Comment:
Can you please wrap the code portion in backticks, then add a full stop after the last backticks?